src/FileUpload/FileUploadStyle.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from 'styled-components'
+
+export const FilePreview = styled.img`
+	max-width: 90%;
+	max-height: 300px;
+	padding: 10px;
+	box-sizing: border-box;
+	margin: 0 auto;
+	border: 1px solid #dedede;
```

Comment:
Shouldn't we use theme for common configuration? :)

Reply:
This will be solved in another PR for all components :)
https://github.com/purple-technology/phoenix-components/tree/feature/theme-provider